packages/vuetify/src/components/VAvatar/VAvatar.ts

```diff
@@ -23,7 +27,6 @@ export default mixins(
       type: [Number, String],
       default: 48,
     },
-    tile: Boolean,
   },
 
   computed: {
```

Comment:
Remove `v-avatar--tile` line from `classes` computed.

Reply:
Done and I also removed the test which focused on v-avatar--tile